docs/spec.rst

```diff
@@ -149,6 +149,19 @@ Following mapping keys are supported (all but the marked once are optional):
         The path in which ``python setup.py install`` will be executed.
         Defaults to the repository root.
 
+    ``version``
```

Comment:
Seems weird there's an example here that doesn't include the actual `version` object that it's documenting.